### DISCUSSION TITLE
LinkerD and LinkerD multilcuster upgrade from 2.9.4 to 2.12.0

### DISCUSSION THREAD

**Discussion by Vamsi0473** [linkerd/linkerd2#9389] [General]
Hi Team, currently we are using linkerd 2.9.4 version in production but in order to leverage failover which is introduced in 2.12.0 stable version we are planning to upgarde to the latest 2.12.0 version can someone shed light here in what are steps that I should follow for upgrading is it something like step upgrade or can I directly upgrade to 2.12.0 from 2.9.4. I tried to check the documentation here https://linkerd.io/2.12/tasks/upgrade/ but I didn't understand how I have to proceed in my use case

**Comment by kleimkuhler** [comment]
Just to clarify, you don't know how to proceed with your use case because it's a much older major version? A direct upgrade may work (2.9 -> 2.12), but you can also fallback to upgrading to each major version (2.9 -> 2.10, 2.10 -> 2.11, and then 2.11 -> 2.12).
